groups/infrastructure/main.tf

```diff
@@ -72,6 +72,32 @@ module "enablement-presenter-api-alb" {
   }
 }
 
+module "ch-service-mock-alb" {
+  source = "git@github.com:companieshouse/terraform-modules//aws/application_load_balancer?ref=1.0.329"
+
+  environment             = var.environment
+  service                 = "ch-service-mock"
+  ssl_certificate_arn     = data.aws_acm_certificate.cert.arn
+  subnet_ids              = split(",", local.subnet_ids_private)
+  vpc_id                  = data.aws_vpc.vpc.id
+  idle_timeout            = 1200
+  create_security_group   = true
+  internal                = true
+  count                   = var.enable_ch_service_mock_alb ? 1 : 0
```

Comment:
Could the count be separated from the other variables and placed at the top so it's obvious?